lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1415,5 +1416,14 @@ def api_client
       end
       @client
     end
+
+    # Encode by UTF-8 with invalid characters replaced by spaces.
+    def coerce_to_utf8_encode(input)
```

Comment:
Sounds like a good idea to me, in case users prefer to crash rather than ingest invalid logs. But, just like the API, we should make this behaviour the default.